.github/workflows/ci.yml

```diff
@@ -9,8 +9,8 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        python-version: ['3.9','3.10','3.11','3.12']
-        aiida-version: ['stable']
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
no 3.13?

Reply:
`pymatgen` requires `"<3.13,>=3.10"`, so `poetry` complains (it would also be an issue for `torch<2.6`, which the currently released `janus-core` asks for).